### PR TITLE
"Math Magicians: events"

### PR DESCRIPTION
**Pull request by AmaliaGM**
## Project requirements

- To help you concentrate on how to work with React events, we have developed the math logic you need to make the Calculator component work. You can use the files [calculate.js](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js) and [operate.js](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js).

- Copy the files calculate.js and operate.js into a logic/ directory in your project.
- Analyze the files calculate.js and operate.js, be sure you understand what they do (note: you need to add [big.js](https://www.npmjs.com/package/big-js) as dependency).
- Import the files in your Calculator component.
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component.
- Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.
- Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

**Comment by ahmetbozaci** [comment]
# ❌ ❌ ❌ STATUS: INVALID  ❌ ❌ ❌ 


### Hello @AmaliaGM  ✋🏼 
#### You have submitted a project, that has linter errors. Kindly fix the errors and submit it for review again.
#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏


### Cheers and Happy coding! 💻 👏
coding

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

_Invalid Code Review Request **does not count** into the code reviews limit._

___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @AmaliaGM , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have ReadMe File you can use [this template](https://github.com/microverseinc/readme-template) as a guide.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AmaliaGM** [comment]
> Hi @AmaliaGM ,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that does not have ReadMe File you can use [this template](https://github.com/microverseinc/readme-template) as a guide.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Hi. Hope you´re well. I´m sorry,  but the project does have a README.md file 
image

**Comment by danielufeli** [comment]
Oops, I meant to say your ReadMe file is not descriptive, kindly use [this template](https://github.com/microverseinc/readme-template) to make it descriptive.

**Comment by AmaliaGM** [comment]
> Oops, I meant to say your ReadMe file is not descriptive, kindly use [this template](https://github.com/microverseinc/readme-template) to make it descriptive.

Allright, Thanks for answering!

File: README.md [hunk 1]
```diff
@@ -1,70 +1,51 @@
-# Getting Started with Create React App
+Math magician
```
Comment (MahmoudBakr23) [inline README.md]: Optional

- In order to have a great README file, there isn't a template that is cooler and more professional than [Microvers's one](https://github.com/microverseinc/readme-template)

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @AmaliaGM 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done working with event handlers to make the operations succeed
- Great work using `useContext`
- Nice code splitting

**Optional**
_Please check the comment under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmaliaGM** [comment]
@MahmoudBakr23 Thanks so much ! And you're so right.. this project came with the README already but I'll change it.. Happy weekend!

**Pull request merged**